grantnav/frontend/templates/funder.html

```diff
@@ -1,194 +1,228 @@
-          <thead>
-            <tr>
-              <th>Date</th>
-              <th class="amount">Amount</th>
-              <th>Recipient</th>
-              <th>Title</th>
-              <th>Description</th>
-            </tr>
-          </thead>
-        </table>
+
+    <div class="section spacer-10">
+        <div class="grantnav-datatable__content--filters">
+          <h4>Grants</h4>
+          <div class="export-wrapper">
+            <div class="export-label">Export search data:</div>
+
+            <div class="export-button">
+              {% include 'components/export-data-button--csv.html' with data_function='search.csv' query_string=query_string %}
+            </div>
+
+            <div class="export-button">
+              {% include 'components/export-data-button--json.html' with data_function='search.json' query_string=query_string %}
+            </div>
+          </div>
+        </div>
+        <div class="table table--zebra">
+          <table class="table table--zebra table--secondary dt-responsive full-width" id="grants_datatable">
+            <thead>
+              <tr>
+                <th>Date</th>
+                <th class="amount">Amount</th>
+                <th>Recipient</th>
+                <th>Title</th>
+                <th>Description</th>
+              </tr>
+            </thead>
+          </table>
+        </div>
       </div>
     </div>
-  </div>
+  </main>
 </div>
-
 {% endblock %}
-
 {% block extra_style %}
-   <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.10/css/dataTables.bootstrap.min.css">
-   <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.0.2/css/responsive.dataTables.min.css">
-   <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/scroller/1.4.1/css/scroller.dataTables.min.css">
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.10/css/dataTables.bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.0.2/css/responsive.dataTables.min.css">
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/scroller/1.4.1/css/scroller.dataTables.min.css">
 {% endblock %}
-
 {% block extra_scripts %}
-   <script type="text/javascript" src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js"></script>
-   <script type="text/javascript" src="https://cdn.datatables.net/1.10.10/js/dataTables.bootstrap.min.js"></script>
-   <script type="text/javascript" src="https://cdn.datatables.net/responsive/2.0.2/js/dataTables.responsive.min.js"></script>
-   <script type="text/javascript" src="https://cdn.datatables.net/scroller/1.4.1/js/dataTables.scroller.min.js"></script>
+<script
+  type="text/javascript"
+  src="https://cdn.datatables.net/1.10.10/js/jquery.dataTables.min.js"
+></script>
+<script
+  type="text/javascript"
+  src="https://cdn.datatables.net/1.10.10/js/dataTables.bootstrap.min.js"
```

Comment:
I think datatables depends on bootstrap, which we don't use any more so some of the icons and table functionality is broken. We might want to replace this with something else, at some point.

Reply:
I'm not seeing this as an issue on the dev site. On the datatables docs it states:

>the following Javascript library files are loaded for use in this [bootstrap-4] example:
>
>https://code.jquery.com/jquery-3.5.1.js
>https://cdn.datatables.net/1.10.24/js/jquery.dataTables.min.js
>https://cdn.datatables.net/1.10.24/js/dataTables.bootstrap4.min.js

So the only direct dependency is jquery which we do have. I we don't need the `dataTables.bootstrap4.min.js` extension as this seems to just insert classes which don't exist in our CSS, styling is currently being provided by the main theme css. 

The sort arrow / filter icons are however being provided by  `dataTables.bootstrap.min.css`

I suggest that we remove the redundant js extension and keep as is.

Reply:
Added f162d29